examples/example_for_updating_metadata.py

```diff
@@ -0,0 +1,115 @@
+    dataset_description_editor.add_values("12L digital twin UUID", row_name='Identifier type')
+    dataset_description_editor.add_values("1", row_name='Number of subjects')
+    dataset_description_editor.add_values("1", row_name='Number of samples')
+
+    # Step4, add values for code_description_editor need to specify field_name
+    code_description_editor.add_values(*["test..1", "test2", "test3", "test4", "test5...", ],
+                                       row_name="TSR1: Define Context Clearly Rating (0-4)", append=False)
+
+    # Step4, add values for code_parameters_editor to add values in a row, append = False
+    code_parameters_editor.add_values(
+        *["breast ...", "test..1", "test2", "test3", "test4", "test5...", "test3", "test4", "test5..."], append=False)
+    # Step4, add values for code_parameters_editor to add values in a row, append = True
+    code_parameters_editor.add_values(
+        *["breast_append", "test1_append", "test2_append", "test3_append", "test4_append", "test5..._append",
+          "test3_append", "test4_append", "test5_append"], append=True)
+    # Step4, add values for code_parameters_editor to add values in a column need to specify field_name, append = True
+    code_parameters_editor.add_values(*["test1_name", "test2_name", "test3_name", "test4_name"], header='name',
+                                      append=True)
+
+    # Step5, get values for code_parameters_editor
+    print(code_parameters_editor.get_values(field_name="name"))
+    # Step5, get values for dataset_description_editor
+    print(dataset_description_editor.get_values(field_name="contributorrole"))
+    # Step5, get values for code_parameters_editor
+    print(code_description_editor.get_values(field_name="TSR1: Define Context Clearly Rating (0-4)"))
+
+    # Step6, remove values in specific header/field_name, code_parameters_editor
+    # code_parameters_editor.remove_values("test1_name", field_name="name")
+
+    # Step6, remove entire values in code_parameters_editor
+    # code_parameters_editor.clear_values()
+    # Step6, remove entire values in dataset_description_editor
+    # dataset_description_editor.clear_values()
+    # Step6, remove entire values in code_description_editor
+    # code_description_editor.clear_values()
+
+    # Step6, remove all values in one specific row/col
+    # code_parameters_editor.clear_values(field_name="name")
+    # dataset_description_editor.clear_values(field_name="Contributor role")
+    # code_description_editor.clear_values(field_name="TSR1: Define Context Clearly Rating (0-4)")
+
+    # Step7, save current dataset
+    dataset.save(save_dir=save_dir)
+
+    # Step8, move files to dataset primary and derivative folder
+    # Copy data from "source_data_raw" to a "sds_dataset" parent directory adhering to SDS framework.
+    dataset.add_data(source_path="./test_data/sample1/raw", subject="subject-xyz", sample="sample-1",
```

Comment:
This could become add_sample()

Reply:
fixed by [2394cf5](https://github.com/SPARC-FAIR-Codeathon/sparc-me/pull/92/commits/2394cf5c2aa697036e27338520944c4a76ebc182)